examples/advanced/stochastic_odes.py

```diff
@@ -59,7 +59,7 @@ def exact_solution(t, dt):
 cols = ceil(1.0 * len(dts) / rows)
 errors = dict([(method, zeros(len(dts))) for method in methods])
 for dt_idx, dt in enumerate(dts):
-    print 'dt: ', dt
+    print('dt:', dt)
```

Comment:
It's not terribly important, but this will look odd in Python 2 because it'll print ``('dt', 0.01*ms)`` rather than ``dt: 0.01*ms``. Same for the other examples.